src/Resources/views/Form/roles_matrix.html.twig

```diff
@@ -0,0 +1,41 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+<table class="table table-condensed">
+    <thead>
+    <tr>
+        <th></th>
```

Comment:
Is this empty one needed?

Reply:
It is needed. Otherwise the table looks not like expected